sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -1,7 +1,54 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2020-02-11)
 
+**Breaking changes**
+
+- The single text, module-level operations return an atomic type of the operation result. For example, `detectLanguage(String text)` returns a `DetectedLanguage` rather than a `DetectLanguageResult`.
+
+  For other module-level operations:
+    
+  `recognizeEntities(String text)`, it no longer returns type of `Mono<RecognizeEntitiesResult>` but `PagedFlux<CategorizedEntity>`.
```

Comment:
we should include a message or something that says that for this methods in the async word, the return changed too